### DISCUSSION TITLE
Option to disable for browser-based editors

### DISCUSSION THREAD

**Discussion by scottrohrig** [gdh1995/vimium-c#969] [Q&A] [answered]
I can't seem to find any info regarding disabling (or forcing vimium to stay in insert mode) while editing in a browser based editor that supports vim keybindings. For example, codesandbox has support for vim keybindings, but pressing escape to exit the editor's insert mode also exits vimium insert mode. How can I either disable vimium for the editor, or force vimium to remain in insert mode while in the editor?

**Comment by gdh1995** [comment, marked as answer]
Um there're some ways to make Vimium C keep in insert mode:
1. add a mapping like `map <a-/> enterInsertMode key="<a-/>"`, and then press `Alt+/` to toggle its insert mode - `Escape` will be passed to pages
2. add `map <a-/> openUrl url="vimium://status/toggle-disabled/^%20<a-/>"`, and then `Alt+/` will toggle Vimium C's "disabled mode"
- 1. the "disabled mode" will disable more features of Vimium C
- 2. this mode affects all frames in a current tab, instead of only one frame
- 3. see https://github.com/gdh1995/vimium-c/wiki/Enable-or-Disable-all-frames-by-a-shortcut
